sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/SpecifyIFormatProviderOrCultureInfo.cs

```diff
@@ -41,10 +42,11 @@ public sealed class SpecifyIFormatProviderOrCultureInfo : SonarDiagnosticAnalyze
             DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(rule);
 
-        private static readonly ImmutableArray<KnownType> formatAndClutureTypes =
+        private static readonly ImmutableArray<KnownType> formatAndCultureType =
             ImmutableArray.Create(
                 KnownType.System_IFormatProvider,
-                KnownType.System_Globalization_CultureInfo
+                KnownType.System_Globalization_CultureInfo,
+                KnownType.System_StringComparison

```

Comment:
needed to pass existing test `"".StartsWith("")`

Reply:
removed it after all, it should be a separate rule (I started a discussion with NicoH)